Mastodon/Scene/Profile/ProfileViewController.swift

```diff
@@ -971,11 +971,7 @@ extension ProfileViewController: DataSourceProvider {
         return nil
     }
     
-    func update(status: MastodonStatus) {
-        assertionFailure("Not required")
-    }
-    
-    func delete(status: MastodonStatus) {
+    func update(status: MastodonStatus, intent: MastodonStatus.UpdateIntent) {
         assertionFailure("Not required")
```

Comment:
We might need to replace this with an actual implementation as posts don't get updated when I change them:

1. Go to my profile page.
2. Tap on one of my statuses (so that I see all its replies etc., you know)
3. Edit that very status
4. The status doesn't get updated on the Profile-screen